### PR TITLE
make it look nicer maybe

### PR DESCRIPTION
**Pull request by nim-ka**
idk test to see if it works because im on mobile

### CODE REVIEW

File: generate-styles.js [hunk 1]
```diff
@@ -25,4 +25,4 @@ names.forEach(function (item) {
 });
 
 console.log(output.slice(0, -2));
-console.log("\{background-color: Crimson;\}");
\ No newline at end of file
+console.log("\{ padding: 5px; background-color: Crimson; color: black; \}");
```
Comment (sticks-stuff) [inline generate-styles.js]: `!important` will be needed to actually make the text colour black

**Comment by sticks-stuff** [comment]
Looks really nice in some places...
image
However it can look really bad in others
image
image

File: generate-styles.js (hunk 1 above)
Comment (nim-ka) [reply generate-styles.js]: oh that's stupid, let me test it now

**Comment by nim-ka** [comment]
try this?

**Comment by sticks-stuff** [comment]
LGTM

**Pull request merged**